recipe/meta.yaml

```diff
@@ -19,8 +19,7 @@ source:
     - patches/0005-Vendor-grpc-1.62.2.patch
 
 build:
-  number: 0
-  skip: true  # [py in (312, 38)]
```

Comment:
I think we still need the python 3.8 skip

Reply:
Yup, it seems you are right. I thought conda-forge dropped 3.8

Reply:
It's imminent, but hasn't happened yet.